demo/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     "skipLibCheck": true
   },
   "include": [
-    "src"
+    "src", "../src/utils/helpers/inject.ts"
```

Comment:
Why is this needed?

Reply:
Must be `tslint fix` command that added this. It is not needed indeed. Thanks!